website/content/docs/connect/gateways/api-gateway/usage.mdx

```diff
@@ -0,0 +1,216 @@
+---
+layout: docs
+page_title: API Gateways on Virtual Machines
+description: Learn how to configure and Consul API gateways and gateway routes on virtual machines so that you can enable ingress requests to services in your service mesh in VM environments.
+---
+
+# API Gateways on Virtual Machines
+
+This topic describes how to deploy Consul API gateways to networks that operate
+in virtual machine (VMs) environments. If you want to implement an API gateway
+in a Kubernetes environment, refer to [API Gateway for Kubernetes](/consul/docs/api-gateway).
+
+## Introduction
+
```

Comment:
```suggestion
Consul API gateways provide a configurable ingress points for requests into a Consul network. Configure the following configuration entries to set up API gateways: 
- [API gateway](/consul/docs/connect/gateways/api-gateway/configuration/api-gateway): Provides an endpoint for requests to enter the network. Define listeners that expose ports on the endpoint for ingress. 
- [HTTP routes](/consul/docs/connect/gateways/api-gateway/configuration/http-route) and [TCP routes](/consul/docs/connect/gateways/api-gateway/configuration/tcp-route): The routes attach to listeners defined in the gateway and control how requests route to services in the network. 
- [Inline certificates](/consul/docs/connect/gateways/api-gateway/configuration/inline-certificate): Makes TLS certificates available to gateways so that requests between the user and the gateway endpoint are encrypted.

You can configure and reuse configuration entries separately. You can define and attach routes and inline certificates to multiple gateways. 
```
Now that I see it, introduction from this page and the content on the main overview page should be swapped.